Upstream-Status: Inappropriate [configuration]

Update context for version 1.3.5.

Signed-off-by: Kai Kang <kai.kang@windriver.com>
---
From dd915e7f70e676aea93f750c75d16ce646e71e4b Mon Sep 17 00:00:00 2001
From: Mark Asselstine <mark.asselstine@windriver.com>
Date: Wed, 9 Sep 2015 11:52:44 -0400
Subject: [PATCH] Revert "build: add $(prefix) to SYSTEMD_UNIT_DIR"

This reverts upstream commit 00e9d6071b2450659ce01bc984f64ecb5d7ba62d
with minor context changes to allow it to apply.

Yocto based builds use /lib/systemd/system for systemd unit files and
since upstream has chosen to use a mechanism for setting
SYSTEMD_UNIT_DIR which is not flexible we have to resort to reverting
this upstream commit to avoid having ${prefix} added to the path.

Signed-off-by: Mark Asselstine <mark.asselstine@windriver.com>
[ywei: rebased to libvirt-1.3.2]
Signed-off-by: Yunguo Wei <yunguo.wei@windriver.com>
---
 daemon/Makefile.am | 2 +-
 src/Makefile.am    | 2 +-
 tools/Makefile.am  | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/daemon/Makefile.am b/daemon/Makefile.am
index 2dbe81b..41ea2db 100644
--- a/daemon/Makefile.am
+++ b/daemon/Makefile.am
@@ -449,7 +449,7 @@ endif ! LIBVIRT_INIT_SCRIPT_UPSTART
 
 if LIBVIRT_INIT_SCRIPT_SYSTEMD
 
-SYSTEMD_UNIT_DIR = $(prefix)/lib/systemd/system
+SYSTEMD_UNIT_DIR = /lib/systemd/system
 BUILT_SOURCES += libvirtd.service
 
 install-init-systemd: install-sysconfig libvirtd.service
diff --git a/src/Makefile.am b/src/Makefile.am
index a316b4d..d271291 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -2633,7 +2633,7 @@ EXTRA_DIST += \
 if WITH_LIBVIRTD
 if LIBVIRT_INIT_SCRIPT_SYSTEMD
 
-SYSTEMD_UNIT_DIR = $(prefix)/lib/systemd/system
+SYSTEMD_UNIT_DIR = /lib/systemd/system
 
 BUILT_SOURCES += virtlockd.service virtlockd.socket \
 		 virtlogd.service virtlogd.socket
diff --git a/tools/Makefile.am b/tools/Makefile.am
index b3227a7..0e58f73 100644
--- a/tools/Makefile.am
+++ b/tools/Makefile.am
@@ -380,7 +380,7 @@ libvirt-guests.init: libvirt-guests.init.in libvirt-guests.sh
 
 
 EXTRA_DIST += libvirt-guests.service.in
-SYSTEMD_UNIT_DIR = $(prefix)/lib/systemd/system
+SYSTEMD_UNIT_DIR = /lib/systemd/system
 
 if LIBVIRT_INIT_SCRIPT_SYSTEMD
 install-systemd: libvirt-guests.service install-sysconfig libvirt-guests.sh
-- 
1.9.1

